packages/go_router/lib/src/delegate.dart

```diff
@@ -16,7 +16,7 @@ import 'typedefs.dart';
 
 /// GoRouter implementation of [RouterDelegate].
 class GoRouterDelegate extends RouterDelegate<RouteMatchList>
-    with PopNavigatorRouterDelegateMixin<RouteMatchList>, ChangeNotifier {
```

Comment:
We missed this when we refactor the pop. PopNavigatorRouterDelegateMixin is no longer needed since we override poproute directly